src/components/dataset/data/DatasetDataView.tsx

```diff
@@ -145,7 +145,7 @@ function DatasetDataView({
           SELECT ${DbColumns.ROW_ID},
             ${selectedTable?.columns?.map((column) => column.name).join(', ')} ${fromClause}
             ${orderProperty ? `ORDER BY ${orderProperty} ${orderDirection}` : ''}
-          LIMIT ${QUERY_LIMIT}`,
+          LIMIT ${rowsPerPage}`,
```

Comment:
Change 2 - We were always querying for 1000 rows, even though we only show up to 250 rows on a given page. [This was why the user was seeing 1000 rows as the total number of rows when there was greater than 1000 rows]

Reply:
EDIT: This change impacted paging - we actually need to remove the limit altogether.